x-pack/plugins/apm/public/components/app/ErrorGroupDetails/view.tsx

```diff
@@ -4,32 +4,34 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { EuiBadge, EuiSpacer, EuiText } from '@elastic/eui';
+import { get } from 'lodash';
 import React, { Fragment } from 'react';
 import styled from 'styled-components';
-import PropTypes from 'prop-types';
-import { get } from 'lodash';
-import { HeaderLarge } from '../../shared/UIComponents';
-import DetailView from './DetailView';
-import Distribution from './Distribution';
-import { KueryBar } from '../../shared/KueryBar';
-
-import { EuiText, EuiBadge, EuiSpacer } from '@elastic/eui';
-import {
-  unit,
-  units,
-  px,
-  colors,
-  fontFamilyCode,
-  fontSizes
-} from '../../../style/variables';
 import {
   ERROR_CULPRIT,
-  ERROR_LOG_MESSAGE,
+  ERROR_EXC_HANDLED,
   ERROR_EXC_MESSAGE,
-  ERROR_EXC_HANDLED
+  ERROR_LOG_MESSAGE
 } from '../../../../common/constants';
 import { ErrorDistributionRequest } from '../../../store/reactReduxRequest/errorDistribution';
 import { ErrorGroupDetailsRequest } from '../../../store/reactReduxRequest/errorGroup';
+import { IUrlParams } from '../../../store/urlParams';
+import {
+  colors,
+  fontFamilyCode,
+  fontSizes,
+  px,
+  unit,
+  units
+} from '../../../style/variables';
+// @ts-ignore
+import { KueryBar } from '../../shared/KueryBar';
+// @ts-ignore
+import { HeaderLarge } from '../../shared/UIComponents';
```

Comment:
This is the last place we use `HeaderLarge`, if you feel like replacing it with `EuiTitle`

Reply:
I do! :)

Reply:
If you want an example of this diff: https://github.com/elastic/kibana/pull/26608/files?utf8=%E2%9C%93&diff=unified&w=1#diff-8e6297b08b3f52c9a4f5ad1ab49f18f3L32 :+1: